scripts/modules/supported-models.json

```diff
@@ -7,7 +7,7 @@
 				"input": 3.0,
 				"output": 15.0
 			},
-			"allowed_roles": ["main", "fallback"],
+			"allowed_roles": ["main", "fallback", "research"],
```

Comment:
out of scope of this PR, please add to separate PR so we can determine if it can be added as a research model or not (if it has web search capabilities)